magento2/.docker/generate-code.sh

```diff
@@ -0,0 +1,11 @@
+#!/bin/bash -x
+
+# Run this from the root dir of the repo with ./.docker/generate-code.sh
+docker run -d -it --name mage_code_gen \
```

Comment:
If the container is already running, why does this start a new one?

Reply:
File removed, this is unnecessary as will be mounted as a volume